spec/services/hashtag_autocomplete_service_spec.rb

```diff
@@ -235,6 +236,38 @@ def self.search_sort(search_results, _)
         expect(subject.search("book", %w[category tag]).map(&:text)).to eq(["Book Club"])
       end
     end
+
+    context "when no term is provided (default results) triggered by a # with no characters in the UI" do
+      fab!(:category2) do
+        Fabricate(:category, name: "Book Zone", slug: "book-zone", topic_count: 546)
+      end
+      fab!(:category3) do
+        Fabricate(:category, name: "Book Dome", slug: "book-dome", topic_count: 987)
+      end
+      fab!(:category4) { Fabricate(:category, name: "Bookworld", slug: "book", topic_count: 56) }
+      fab!(:category5) { Fabricate(:category, name: "Media", slug: "media", topic_count: 446) }
+      fab!(:tag2) { Fabricate(:tag, name: "mid-books", topic_count: 33) }
+      fab!(:tag3) { Fabricate(:tag, name: "terrible-books", topic_count: 2) }
+      fab!(:tag4) { Fabricate(:tag, name: "book", topic_count: 1) }
+
+      it "returns the N top categories and tags (based on topic_count) that the user can access" do
```

Comment:
Just to clarify, what does `N` represent here?

Reply:
Changing this, N is irrelevant here, thanks for pointing this out